src/Search/Captures/SearchManipulateCapture_MySQLDatabase.php

```diff
@@ -5,6 +5,10 @@
 use SilverStripe\ORM\Connect\MySQLDatabase;
 use SilverStripe\FullTextSearch\Search\Updaters\SearchUpdater;
 
+/**
+ * @deprecated since version 3.1.0
```

Comment:
Can you add a message suggesting a replacement, e.g. `@deprecated 3.1...4.0 Please use tractorcow/proxydb to proxy the database connector instead`